packages/common/src/helper/formatDate.ts

```diff
@@ -0,0 +1,15 @@
+export const formatDate = (value: string) => {
+  if (!value) {
+    return;
+  }
+
+  const date = new Date(value);
+
+  // Date object subtracts a day, so need to re-add by 1
+  const day = parseInt(date.toLocaleString('default', { day: '2-digit' })) + 1;
+
+  const month = date.toLocaleString('default', { month: 'short' });
+
+  const year = date.toLocaleString('default', { year: 'numeric' });
+  return `${month} ${day}, ${year}`;
+};
```

Comment:
Nit - It might be good to use a date library like `DayJs` to do this sort of thing.